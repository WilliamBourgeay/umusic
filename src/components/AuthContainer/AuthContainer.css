@keyframes animation {
  0%   {
    transform: translateX(-10%);
    opacity: 0;
  }
  100% {
    transform: translateX(0%);
    opacity: 1;
  }
}

@media (min-height: 400px) and (min-width: 400px){
  .Wrapper {
    animation-name: animation;
    animation-timing-function: ease-out;
    animation-duration: 0.7s;
    animation-fill-mode: both;
    margin-left: 15%;
    margin-right: auto;
    margin-top: 30vh;
    transform: translateY(-30%);
    height: auto;
    width: 25%;
    background-color: rgba(238, 238, 238, 0.9);
    border-radius: 7px;
    box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.2);
    padding: 5px 20px 20px 20px;
    overflow-y: auto;
  }
}

@media (max-height: 400px) {
  .Wrapper {
    left: 0;
    top: 0;
    height: 100vh;
    background-color: rgba(238, 238, 238, 0.9);
    padding: 5px 20px 20px 20px;
    overflow-y: auto;
  }
}
@media (max-width: 400px) {
  .Wrapper {
    left: 0;
    top: 0;
    height: 100vh;
    background-color: rgba(238, 238, 238, 0.9);
    padding: 5px 20px 20px 20px;
    overflow-y: auto;
  }
}
