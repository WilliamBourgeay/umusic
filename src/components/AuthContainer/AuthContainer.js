// @flow

import * as React from 'react';
import './AuthContainer.css';

type Props = {
  children: React.Node,
}

const AuthContainer = ({ children }: Props) => (
  <div className="AuthContainer">
    {children}
  </div>
);

export default AuthContainer;
