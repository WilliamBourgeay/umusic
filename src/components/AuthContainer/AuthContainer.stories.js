import React from 'react';
import { storiesOf } from '@storybook/react';
import TextField from '@material-ui/core/TextField';
import AuthContainer from './AuthContainer';

const stories = storiesOf('AuthContainer', module);

stories
  .add('AuthContainer', () => (
    <AuthContainer>
      <div>
        <TextField
          id="name"
          label="name"
          type="default"
          margin="normal"
        />
        <TextField
          id="email"
          label="email"
          type="default"
          margin="normal"
        />
      </div>
    </AuthContainer>
  ));
