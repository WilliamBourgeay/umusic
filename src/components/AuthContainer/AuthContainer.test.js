import * as React from 'react';
import renderer from 'react-test-renderer';

import AuthContainer from './AuthContainer';

describe('AuthContainer', () => {
  it('should render a AuthContainer', () => {
    const tree = renderer.create(
      <AuthContainer>
        <div />
      </AuthContainer>,
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
