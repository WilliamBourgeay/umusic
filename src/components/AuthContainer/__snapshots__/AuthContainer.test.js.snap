// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthContainer should render a AuthContainer 1`] = `
<div
  className="AuthContainer"
>
  <div />
</div>
`;
